pipeline/azure-pipelines.yml

```diff
@@ -0,0 +1,244 @@
+      vsVersion: 17.0
+      msbuildArgs: /p:VsTargetVersion=$(vsTargetVersion) /p:DeployExtension=false /p:SignArtifacts=$(SHOULD_SIGN) /p:AssemblyOriginatorKeyFile="$(snk.secureFilePath)" /p:pfxCertificatePath="$(pfx.secureFilePath)" /p:pfxPassword="$(pfxPassword2021-2023)" /p:pfxSha1=$(pfxSha1-2021-2023) /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId)  $(AdditionalMSBuildArgs)
+      platform: $(BuildPlatform)
+      configuration: $(BuildConfiguration)
+      maximumCpuCount: true
+      logFileVerbosity: diagnostic
+
+  - task: UniversalPackages@0
+    displayName: Download WhiteSource agent
+    condition: and(succeeded(), or(eq(variables['ForceWhiteSourceScan'], 'true'), eq(variables['Build.SourceBranch'], 'refs/heads/master')))
+    inputs:
+      downloadDirectory: $(System.DefaultWorkingDirectory)\Whitesource
+      feedListDownload: 399fb241-ecc7-4802-8697-dcdd01fbb832/69e11f1b-cad7-442a-b676-37722e207221
+      packageListDownload: agent
+      versionListDownload: '*'
+
+  - task: PowerShell@2
+    displayName: Run WhiteSource (Mend) scan
+    condition: and(succeeded(), or(eq(variables['ForceWhiteSourceScan'], 'true'), eq(variables['Build.SourceBranch'], 'refs/heads/master')))
+    inputs:
+      targetType: inline
+      script: java -jar  "${env:SYSTEM_DEFAULTWORKINGDIRECTORY}\Whitesource\wss-unified-agent.jar" -c "${env:BUILD_SOURCESDIRECTORY}\build\whitesource\wss-unified-agent.config" -product "SonarLint/VisualStudio"  -project "SonarSource/sonarlint-visualstudio VS$(vsTargetVersion) ${env:SONAR_PROJECT_VERSION}" -projectversion ${env:SONAR_PROJECT_VERSION}  -scanComment "buildNumber:${env:BUILD_BUILDID};gitSha:${env:BUILD_SOURCEVERSION}" -offline false
+
+  - task: DotNetCoreCLI@2
+    displayName: Install cycloneDX tool
+    condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'master'))
+    inputs:
+      command: custom
+      custom: tool
+      arguments: install --global CycloneDX
+
+  - task: DotNetCoreCLI@2
+    displayName: Generate cycloneDX sbom file
+    condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'master'))
+    inputs:
+      command: custom
+      projects: SonarLint.VisualStudio.Integration.sln
+      custom: CycloneDX
+      arguments: -t -j -o binaries
+  - task: DownloadSecureFile@1
+    name: signKey
+    displayName: Download sign key
+    condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'master'))
+    inputs:
+      secureFile: 3755ac9b-4269-4218-a8d7-97aa4f1dd117
+
+  - task: PowerShell@2
+    displayName: Rename and sign SBOM file
+    condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'master'))
+    inputs:
+      targetType: filePath
+      filePath: $(System.DefaultWorkingDirectory)\pipeline\scripts\sbom.ps1
+
+  - task: CopyFiles@2
+    displayName: 'Copy VSIX to: $(Build.ArtifactStagingDirectory)'
+    inputs:
+      SourceFolder: $(Build.SourcesDirectory)\binaries
+      Contents: >-
```

Comment:
I'm guessing the extra line-feed between the line settings required (i.e. line 186) is required here because you are using the [block chomping syntax](https://yaml-multiline.info/) `>-`.

It might be simpler to use the literal syntax and get rid of the extra blank line i.e. 
```
    Contents: |
     **\*.vsix
     **\SonarLint.visualstudio.sbom*.*
```